sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/implementation/Utility.java

```diff
@@ -26,7 +27,7 @@ private Utility() {
 
     /**
      * Automatically detect byte buffer's content type.
-     *
+     * <p>
```

Comment:
missing closing </p>

Reply:
Javadoc is formatted correctly even without closing tags https://stackoverflow.com/questions/33787753/javadoc-closing-tags